ArjunaJTA/jta/src/test/resources/standalone-cmr.xml

```diff
@@ -148,6 +148,14 @@
                         <password>sa</password>
                     </security>
                 </datasource>
+                <datasource jndi-name="java:jboss/datasources/ExampleDS2" pool-name="ExampleDS2" enabled="true" use-java-context="true">
```

Comment:
I think there is missing attribute `connectable=true`. If so then it will be missing in the `ExampleDS` as well.

Reply:
You are correct. I have added to both datasources (the integration test appears to use CMR both with and without this property).